script/gh/docker-tags.rb

```diff
@@ -17,24 +17,6 @@ def latest_release_major_minor
   nil
 end
 
-def generate_matrix
-  latest_version = latest_release_major_minor
```

Comment:
The method `latest_release_major_minor` can also be removed, as well as `net/http` and `json` imports.